app/Models/UsersActions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class UsersActions extends Model
```

Comment:
please rename it to 
```suggestion
class UserActions extends Model
```
models should be singular because it is one entity
see guidance: https://github.com/alexeymezenin/laravel-best-practices#follow-laravel-naming-conventions